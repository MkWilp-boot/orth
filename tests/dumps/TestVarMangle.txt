ORTH_ERR_03: Redeclaration of "Var" -> "teste" in block "main"