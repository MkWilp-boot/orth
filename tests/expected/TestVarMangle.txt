ORTH_ERR_03: Redeclaration of "Var" -> "space" in block "main"